src/toast/tests/ops_sim_tod_conviqt.py

```diff
@@ -282,6 +282,89 @@ def test_sim(self):
 
         return
 
+    def test_sim_TEB(self):
+        if not ops.conviqt.available():
+            print("libconviqt not available, skipping tests")
+            return
+
+        # Create a fake scan strategy that hits every pixel once.
+        data = create_healpix_ring_satellite(self.comm, nside=self.nside)
+
+        # Generate timestreams
+
+        detpointing = ops.PointingDetectorSimple()
+
+        key = defaults.det_data
+
+        sim_conviqt = ops.SimTEBConviqt(
+            comm=self.comm,
+            detector_pointing=detpointing,
+            sky_file=self.fname_sky,
+            beam_file=self.fname_beam,
+            dxx=False,
+            det_data=key,
+            normalize_beam=True,
+            fwhm=self.fwhm_sky,
+        )
+        sim_conviqt.apply(data)
+
+        # Bin a map to study
+
+        pixels = ops.PixelsHealpix(
+            nside=self.nside,
+            nest=False,
+            detector_pointing=detpointing,
+        )
+        pixels.apply(data)
+        weights = ops.StokesWeights(
+            mode="I",
+            detector_pointing=detpointing,
+        )
+        weights.apply(data)
+
+        default_model = ops.DefaultNoiseModel()
+        default_model.apply(data)
+
+        cov_and_hits = ops.CovarianceAndHits(
+            pixel_dist="pixel_dist",
+            pixel_pointing=pixels,
+            stokes_weights=weights,
+            noise_model=default_model.noise_model,
+            rcond_threshold=1.0e-6,
+            sync_type="alltoallv",
+        )
+        cov_and_hits.apply(data)
+
+        binner = ops.BinMap(
+            pixel_dist="pixel_dist",
+            covariance=cov_and_hits.covariance,
+            det_data=key,
+            det_flags=None,
+            pixel_pointing=pixels,
+            stokes_weights=weights,
+            noise_model=default_model.noise_model,
+            sync_type="alltoallv",
+        )
+        binner.apply(data)
+
+        # Study the map on the root process
+
+        toast_bin_path = os.path.join(self.outdir, "toast_bin.fits")
```

Comment:
I would feel more confident if this version of the operator was compared to another version in a case where we expect identical results.

Reply:
sounds good ! will implment a test

Reply:
The tests have been reviewed there are major changes involving mainly the scanning strategy (to solve for QU pixels we need more >3 hits), the way the input beam is simulated. 
See  (https://github.com/hpc4cmb/toast/blob/aa03ca92dad37a57a5c5f6afabd65fb61bdc9787/src/toast/tests/ops_sim_tod_conviqt.py#L35)

Reply:
The operators `SimWeightedConviqt` and `SimTEBConviqt` are tested and compared with `SimConviqt` as they are expect to produce exactly the same convolution outputs in the absence of  spinning HWP